src/Hungary/Mews.Fiscalizations.Hungary/Models/Invoice/Receiver.cs

```diff
@@ -15,7 +15,7 @@ public Receiver(Company company)
         {
         }
 
-        public static ITry<Receiver, Error> LocalCompany(TaxpayerIdentificationNumber taxpayerId, Name name, SimpleAddress address)
+        public static ITry<Receiver, Error> LocalCompany(string taxpayerId, Name name, SimpleAddress address)
```

Comment:
Does this simplify the usage?

Reply:
Yea it should as you don't have to validate the tax id and then the local company
there is an example in the modified tests